manifester/manifester.py

```diff
@@ -48,6 +52,24 @@ def access_token(self):
             ).json()
             self._access_token = token_data["access_token"]
         return self._access_token
+    
+    @property
```

Comment:
likely could be a cached_property

Reply:
Could you explain the distinction? I have not used a cached_property previously.

Reply:
I looked this up and agree with you - a new value will be added approximately every 6 months, so the valid versions will likely never change during a manifester run.